src/swap/components/Swap.tsx

```diff
@@ -139,8 +139,8 @@ export function Swap({ address, children, title = 'Swap' }: SwapReact) {
         return;
       }
       try {
```

Comment:
@abcrane123 follow up PR. Same here, add line comments to explain what's going on.